shoov/modules/custom/shoov_restful/plugins/restful/node/ci_build_status/1.0/ShoovCiBuildStatusResource.class.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/**
+ * @file
+ * Contains ShoovCiBuildStatusResource.
+ */
+
+class ShoovCiBuildStatusResource extends \RestfulEntityBase {
+
+  /**
+   * Overrides \RestfulDataProviderEFQ::controllersInfo().
+   *
+   * Accept only GET request with an id.
+   */
+  public static function controllersInfo() {
+    return array(
+      '^.*$' => array(
+        \RestfulInterface::GET => 'viewEntities',
+      ),
+    );
+  }
+
+  /**
+   * Overrides \RestfulEntityBase::publicFieldsInfo().
+   */
+  public function publicFieldsInfo() {
+    $public_fields = array();
+
+    $public_fields['status'] = array(
+      'callback' => array($this, 'statusHtml'),
```

Comment:
Can't change callback to `process_callbacks` because default value for the status is `NULL` and in this case process callback is not running.